use-timescale/security/transit-gateway.md

```diff
@@ -0,0 +1,86 @@
+---
+
+# Peer your $SERVICE_LONGs with AWS Transit Gateway
+
+You use [AWS Transit Gateway][aws-transit-gateway] as a traffic controller for your network. Instead of setting up lots of direct connections to virtual private clouds, on-premise data centers, and other AWS services, you connect everything to Transit Gateway. This simplifies your network and makes it easier to manage and scale.
+
+$CLOUD_LONG allows you to then create a peering connection between your $SERVICE_SHORTs and AWS Transit Gateway. This means that, no matter how big or complex your infrastructure is, you can connect securely to your $SERVICE_LONGs. 
+
+To configure this secure connection, you:
+
+1. Create a $CLOUD_LONG Peering $VPC with a peering connection to your AWS Transit Gateway.
+1. Accept and configure the peering connection on your side.
+1. Attach individual $SERVICE_SHORTs to the Peering $VPC.
+
+## Create a Peering $VPC
+
+To create a Peering $VPC:
+
+<Procedure>
+
+1. **In [$CONSOLE][console-login] > `Security` > `VPC`, click `Create a VPC`**
+
+   ![$CLOUD_LONG new $VPC](https://assets.timescale.com/docs/images/add-peering-vpc.png)
+
+1.  **Choose your region and IP range, name your VPC, then click `Create VPC`**
+
+    ![Create a new VPC in $CLOUD_LONG](https://assets.timescale.com/docs/images/configure-peering-vpc.png)
+
+    Your $SERVICE_SHORT and Peering $VPC must be in the same AWS region. The number of Peering $VPCs you can create in your project depends on your [pricing plan][pricing-plans]. If you need another Peering $VPC, either contact [support@timescale.com](mailto:support@timescale.com) or change your pricing plan in [$CONSOLE][console-login].
+
+1.  **Add a peering connection**
+
+    1. In the `VPC Peering` column, click `Add`.
+    1. Provide your AWS account ID, VPC ID or Transit Gateway ID, optionally CIDR range, and AWS region. 
+    1. Click `Add connection`.
+
+    ![Add peering](https://assets.timescale.com/docs/images/add-peering.png)
+
+</Procedure>
+
+## Accept and configure peering connection
+
+Once your peering connection appears as `Processing`, you can accept and configure it on the Transit Gateway side: 
+
+<Procedure>
+
+1. **Accept the peering request**
+
+   In your AWS account, accept the peering request coming from the $COMPANY AWS account. The peering request can take up to 5 min to arrive. Once accepted, the peering should appear as `Connected` in $CONSOLE.
+
+1. **Configure networking in your AWS account**
+
+    Configure at least the following:
+
+    1. Your subnet route table to route traffic to your Transit Gateway for the Peering VPC CIDRs.
+    1. Your Transit Gateway route table to route traffic to the newly created Transit Gateway peering attachment for the Peering VPC CIDRs.
+    1. Security groups to allow outbound TCP 5432.
+
+</Procedure>
+
+## Attach a $CLOUD_LONG service to the Peering VPC
+
+<Procedure>
+
+1. In $CONSOLE > Services, select the $SERVICE_SHORT you want to connect to the Peering VPC.
```

Comment:
Can you highlight to match the next steps, and possibly add the */services link for $CONSOLE please.